lib/render/RayCaster.cpp

```diff
@@ -98,6 +112,16 @@ RayCaster::~RayCaster()
         glDeleteTextures( 1, &_colorMapTextureId );
         _colorMapTextureId = 0;
     }
+    if( _xyCoordsTextureId )
+    {
+        glDeleteTextures( 1, &_xyCoordsTextureId );
+        _xyCoordsTextureId = 0;
+    }
+    if( _zCoordsTextureId )
+    {
+        glDeleteTextures( 1, &_zCoordsTextureId );
+        _zCoordsTextureId = 0;
+    }
 
     // delete buffers
     if( _frameBufferId )
```

Comment:
You call glDeleteBuffers on a FrameBuffer object which causes a memory error in OpenGL.

You need to use glDeleteFramebuffers.

Reply:
Fixed, thansk!